rero_ils/modules/holdings/api_views.py

```diff
@@ -43,3 +44,38 @@ def holding_availability(holding_pid):
     return jsonify({
         'availability': holding.available
     })
+
+
+@api_blueprint.route('/<holding_pid>/pattern/preview/', methods=['GET'])
+@check_authentication
+@jsonify_error
+def pattern_preview(holding_pid):
+    """HTTP GET request for holdings pattern preview.
+
+    Required parameters: holding_pid
+    Optional parameters: size: number of previewed issues - by default 10
+    """
+    try:
+        size = flask_request.args.get('size')
```

Comment:
you can rewrite this as ` number_issues = int(flask_request.args.get('size'), 10)`